### ISSUE TITLE
Error in 0.3.0 when trying to load dbt_project.yml on Windows

### ISSUE THREAD

**Issue by camfrout** [innoverio/vscode-dbt-power-user#25]
Hi!

I am getting an error after the updated to version 0.3.0.
This time I am using Windows and dbt 0.18.0.

```
We should never come here, means that our exceptions are not handled! Error: ENOENT: no such file or directory, open '\c:\Users\xxxx\xxxx\xxxx\dbt_project.yml'
	at Object.openSync (fs.js:447:3)
	at Object.func (electron/js2c/asar.js:140:31)
	at Object.func [as openSync] (electron/js2c/asar.js:140:31)
	at Object.readFileSync (fs.js:349:35)
	at Object.fs.readFileSync (electron/js2c/asar.js:542:40)
	at Object.fs.readFileSync (electron/js2c/asar.js:542:40)
	at x.readAndParseProjectConfig (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:8971)
	at x.<anonymous> (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:9148)
	at Generator.next (<anonymous>)
	at c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:7550
	at new Promise (<anonymous>)
	at n (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:7295)
	at x.refresh (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:9067)
	at x.<anonymous> (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:8823)
	at Generator.next (<anonymous>)
	at c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:7550
	at new Promise (<anonymous>)
	at n (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:7295)
	at x.tryRefresh (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:8774)
	at c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:3133
	at Map.forEach (<anonymous>)
	at Object.<anonymous> (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:3119)
	at Generator.next (<anonymous>)
	at c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:2255
	at new Promise (<anonymous>)
	at n (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:2000)
	at Object.tryRefreshAll (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:3064)
	at c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:16558
	at Generator.next (<anonymous>)
	at a (c:\Users\xxxx\.vscode\extensions\innoverio.vscode-dbt-power-user-0.3.0\dist\extension.js:1:14577)
	at processTicksAndRejections (internal/process/task_queues.js:85:5)
```

If I do a `cat '\c:\Users\xxxx\xxxx\xxxx\dbt_project.yml'` in my terminal I also get an error, but if I do a `cat 'c:\Users\xxxx\xxxx\xxxx\dbt_project.yml'`, it reads the file. So my guess is that the first `\` should be removed for the extension to work again on Windows.

Cheers,

**Comment by mdesmet** [comment]
I removed the cd command as there is an option to set the cwd when launching the terminal.

Also we will always open a new terminal as potentially the path of the existing terminal might be wrong. When your venv is correctly setup through vscode-python extension, it should execute the command after the venv initialization.

Can you let us know if it works for you?

**Comment by camfrout** [comment]
Hi! Thanks for the quick fix.

There is no longer an error on `dbt_project.yml` but I have the same error for `manifest.json` now.

```
File not found at '\c:\xxxxxxxx\target\manifest.json', probably not compiled! Error: ENOENT: no such file or directory, open '\c:\xxxxxxxx\target\manifest.json'
	at Object.openSync (fs.js:440:3)
	at Object.func [as openSync] (electron/js2c/asar.js:140:31)
	at Object.readFileSync (fs.js:342:35)
	at Object.fs.readFileSync (electron/js2c/asar.js:542:40)
	at l.readAndParseManifest 
```

**Comment by mdesmet** [comment]
We missed that one on the manifest. Fixed in 0.3.3.

**Comment by camfrout** [comment]
Using 0.3.5 now and it is fixed. Thanks!
